docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1391,25 +1403,39 @@ Summary of Iron Intervention Algorithm
   mother_iron_group   = ('baseline' if mother_fortified_baseline_i else
                          'intervention_not_baseline' if mother_fortified_i else
                          'uncovered')
-  bw_shift_i = bw_response * iron_concentration * daily_flour_i
-  birth_weight_i      = birthweight_gbd_i + (
-                        -baseline_coverage(t) + mother_fortified_i
-                        ) * bw_shift_i
-
-  # At each simulation time step (including when simulant is born)
-  child_fortified_i(t)  = time_covered_i <= t
-                        = p_i < coverage(t)
-    # If child is covered in baseline, then time_covered_i = float('-inf'),
-    # and hence hb_lag_fraction(t-time_covered_i) = 1, i.e. the child receives
-    # the full age-dependent effect in this case since the time lag has passed.
-  hb_i(t)               = hb_gbd(age_i(t)) + (
-                          -baseline_coverage(t) + child_fortified_i(t) * hb_lag_fraction(t-time_covered_i)
-                          ) * hb_shift *  hb_age_fraction(age_i(t))
+    # Effect on birthweight
+  daily_flour_i       = average daily flour consumption Y of simulant's mother
```

Comment:
if `mother_fortified_i = 1`?

Reply:
Well, we only use it if `mother_fortified_i == True`, but the variable makes sense for all simulants, and we could actually assign its value if we wanted: It would be distributed according to Y if the mother eats flour (i.e. p_i<c, where p_i is the propensity for unfortified flour and c is the proportion of the population who eats flour, from our coverage data table), and it would be 0 otherwise.

So I _defined_ `daily_flour_i` as the mother's daily flour consumption for all simulants, but I only _assigned it a numerical value_ if `mother_fortified_i == True`, and otherwise assigned it the value `'irrelevant'` since we don't use it in that case.

Is that confusing?

Reply:
I made a suggested change below. If the suggested change is correct then I think I understand, but if not then I will need to revist.

Reply:
Yes, your suggested change is correct, but I'm avoiding explicitly writing `== True` for boolean variables (unless the SWEs think that would be clearer).

Reply:
Ok, gotcha. Wasn't initially clear to me because I tend to default to 0/1 rather than True/False, but I see now

Reply:
I edited this to separate out the down-shift and put the code relevant to the up-shift into a single `if` block. I think that might make it clearer (though I still didn't explicitly write `==True` or `==1`).